src/lib/utils/contractUtils.js

```diff
@@ -0,0 +1,24 @@
+import  Contract from 'stellar-sdk';
+
+export async function generateContractClient(contractId, server) {
+    try {
+        // Validate contract ID format
+        if (!contractId.match(/^C[A-Z2-7]{55}$/)) {
```

Comment:
`@stellar/stellar-sdk` provides a helper function for this: https://stellar.github.io/js-stellar-sdk/StrKey.html#.isValidContract

Reply:
used as indicated ser